app/code/Magento/InventoryInStorePickupAdminUi/Controller/Adminhtml/Order/NotifyPickup.php

```diff
@@ -69,34 +69,31 @@ public function __construct(
      * Notify customer by email
      *
      * @return ResultInterface
-     * @throws InputException
-     * @throws LocalizedException
-     * @throws NoSuchEntityException
      */
     public function execute(): ResultInterface
     {
-        $order = $this->initOrder();
-
-        if ($order) {
-            try {
-                $this->notifyOrderIsReadyForPickup->execute((int)$order->getEntityId());
-                $this->messageManager->addSuccessMessage(__('The customer have been notified and shipment created.'));
-            } catch (LocalizedException $e) {
-                $this->messageManager->addErrorMessage($e->getMessage());
-            } catch (Exception $e) {
-                $this->messageManager->addErrorMessage(__('We can\'t notify the customer right now.'));
-                $this->logger->critical($e);
-            }
+        try {
+            $order = $this->initOrder();
+        } catch (Throwable $e) {
```

Comment:
I'd suggest letting default Magento error processor to handle engine exceptions and avoid processing `Throwable` at this level